tools/bpftool/bpftool.vcxproj

```diff
@@ -154,6 +154,14 @@
       <Project>{75fe223a-3e45-4b0e-a2e8-04285e52e440}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <ResourceCompile Include="..\..\resource\ebpf_resource.rc">
+      <AdditionalIncludeDirectories Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">$(SolutionDir)include</AdditionalIncludeDirectories>
```

Comment:
It'd be nice if the `<AdditionalIncludeDirectories>` was just specified once, instead of once per config/platform since it's the same for all of them anyway, just to have less MSBuild to read and maintain